tests-beta/android/MASVS-NETWORK/MASTG-TEST-0242.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Missing Certificate Pinning in Network Traffic
+platform: network
+id: MASTG-TEST-0x242
+type: [static]
+weakness: MASWE-0047
+---
+
+## Overview
+
+There are various ways how certificate pinning can be done for an application.
+
+Since statically finding all of the locations where certificate pinning is performed might not be feasible, this test case uses dynamic analysis to observe all connections the app makes.
+
+The goal of this test case is to dynamically check if the connection to a server can be intercepted using a [Man-in-the-Middle attack]("../../../Document/0x04f-Testing-Network-Communication.md#mitm-attack). If this is possible, it means that the certificate is not pinned correctly or not pinned at all.
```

Comment:
```suggestion
The goal of this test case is to dynamically check if the connection to a server can be intercepted using a [Machine-in-the-Middle attack]("../../../Document/0x04f-Testing-Network-Communication.md#mitm-attack). If this is possible, it means that the certificate is not pinned correctly or not pinned at all.
```

Reply:
@sushi2k What is the reason for this change proposal? Imo, "MITM" is a defined term, which I don't think we should redefine?